common/djangoapps/track/tests/test_middleware.py

```diff
@@ -76,7 +76,7 @@ def test_default_request_context(self):
         context = self.get_context_for_path('/courses/')
         assert context == {'accept_language': '', 'referer': '', 'user_id': '', 'session': '', 'username': '',
                            'ip': '127.0.0.1', 'host': 'testserver', 'agent': '', 'path': '/courses/', 'org_id': '',
-                           'course_id': '', 'client_id': None}
+                           'course_id': '', 'client_id': None, 'enterprise_uuid': ''}
```

Comment:
[style nitpick] our [guidelines](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#breaking-long-lines) on breaking up long lines refers more to function params but I feel could be applied here. Something like:
```
        assert context == {
            'accept_language': '', 'referer': '', 'user_id': '', 'session': '', 'username': '', 'ip': '127.0.0.1',
            'host': 'testserver', 'agent': '', 'path': '/courses/', 'org_id': '', 'course_id': '', 'client_id': None, 
            'enterprise_uuid': ''
        }
``` 
or where key/values are on individual indented line.

Reply:
@alex-sheehan-edx It's Done